irohad/model/generators/impl/query_generator.cpp

```diff
@@ -83,7 +83,7 @@ namespace iroha {
         return query;
       }
 
-      optional_ptr<GetAccountTransactions>
+      std::shared_ptr<GetAccountTransactions>
```

Comment:
Same question about changing the interface for raw usage

Reply:
Answered in https://github.com/hyperledger/iroha/pull/830#discussion_r173097242